packages/frontend/src/main/components/viewer/ObjectSelection.vue

```diff
@@ -120,26 +121,30 @@ export default {
       })
     },
     isolated() {
+      // TODO
```

Comment:
there are many TODOs in this PR, some of which are probably valid and some seem like comments that were forgotten about and should be deleted, so it would be nice if youd go over the comments & todos left in this PR and get rid of the unnecessary ones